### PR TITLE
Fix/uppsf 1181 add runbook

### PR DESCRIPTION
**Pull request by dbelev**
# Description

## What

Please be specific and try to describe your thought process. State the obvious, since this might be the first time the reviewer is looking at the code

## Why

Copy (if there is one) the text of the original Trello/JIRA ticket in here, with a link back to it for the curious.

## Anything, in particular, you'd like to highlight to reviewers

Mention here sections of code which you would like reviewers to pay extra attention to .E.g

_Would appreciate a second pair of eyes on the test_  
_I am not quite sure how this bit works_  
_Is there a better library for doing x_  

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: .github/runbooks.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  # defaults to `master`
+  updateBranch: master
+
+  ### UPDATING BIZ-OPS REQUIRES EACH RUNBOOK TO BE TIED TO A VALID SYSTEM CODE
+  # Mappings between paths and system codes
+  # unless a custom mapping is specified here
+  # runbook.md tries to parse the system code from the runbook's filename (format: my-sys-code_runbook.md)
+  systemCodes:
+    # paths are relative to root, omitting ./ (case-insensitive)
+    content-search-api-port: runbooks/runbook.md
```
Comment (ivan-p-nikolov) [inline .github/runbooks.yml:38]: The system code is wrong here. It should be `upp-relations-api`

Comment (dbelev) [reply .github/runbooks.yml]: Fixed system code

File: runbooks/runbook.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+content
+
+## Known About By
+
+- dimitar.terziev
+- hristo.georgiev
+- elitsa.pavlova
+- elina.kaneva
+- georgi.kazakov
```
Comment (ivan-p-nikolov) [inline runbooks/runbook.md:31]: Given Histo's email on Friday, we could get away with this edit as well.
```suggestion
```

Comment (dbelev) [reply runbooks/runbook.md]: Removed

**Pull request merged**